grafeas-charts/Dockerfile

```diff
@@ -22,9 +22,9 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}
     tar -zxvf helm-${HELM_VERSION}-linux-amd64.tar.gz && \
     mv linux-amd64/helm /usr/local/bin/helm
 
-WORKDIR /go/src/github.com/grafeas/grafeas/grafeas-charts
-
 COPY . .
 
-RUN chmod +x helm_release.sh
+RUN echo $PWD
```

Comment:
These look like debugging commands - are they still needed?

Reply:
Yes, I'm trying to figure out why it's not building correctly on CloudBuild, hence added these debugging commands. I'll remove them once I resolve the issue.